test/test_pointevaluation.jl

```diff
@@ -129,10 +129,10 @@ end
 function dofhandler2()
     # Computes the L2 projection of a quadratic field exactly
     # but not using L2Projector since we want the DofHandler dofs
-    mesh = generate_grid(Quadrilateral, (20, 20))
-    ip_f = Lagrange{RefQuadrilateral,2}()
-    ip_f_v = ip_f^2
-    qr = QuadratureRule{RefQuadrilateral}(3)
+    mesh = generate_grid(Hexahedron, (10, 10, 10))
```

Comment:
We should still test 2D as well. Perhaps you can make the dimension an input variable to the function (`dofhandler2`) and call it once for 2D and once for 3D?

Reply:
Thanks for the suggestion. I added a boolean parameter into `dofhandler2` to test both cases.